runtimes/alibi-explain/tests/test_white_box_sklearn.py

```diff
@@ -0,0 +1,59 @@
+import json
+from typing import Tuple
+
+import numpy as np
+from pytest_cases import fixture, parametrize_with_cases
+from alibi.api.interfaces import Explainer
+from numpy.testing import assert_array_almost_equal
+
+from mlserver.codecs import NumpyCodec
+from mlserver.types import InferenceRequest
+from mlserver.settings import ModelSettings
+from mlserver_alibi_explain import AlibiExplainRuntime
+from mlserver_alibi_explain.common import convert_from_bytes
+
+EXPLANATION_KEY = {  # Explanation key to compare for each explainer type
+    'tree_shap': 'shap_values',
+    'tree_partial_dependence': 'pd_values',
+}
+
+
+@fixture
+@parametrize_with_cases("model_settings, explainer, request, explain_kwargs")
+async def test_case(model_settings: ModelSettings, explainer: Explainer,
+                    request: InferenceRequest, explain_kwargs: dict) \
+        -> Tuple[AlibiExplainRuntime, Explainer, InferenceRequest, dict]:
+    """
+    Fixture that unpacks the test case into a tuple of the explainer runtime, explainer object, inference request
+    and explain kwargs. The pytest-cases doesn't work too well yet with AsyncIO, therefore we need to treat the
+    fixture as an Awaitable and await it in the tests (See https://github.com/smarie/python-pytest-cases/issues/286).
+    The `explainer`, `request` and `explain_kwargs` are simply passed through.
+    """
+    runtime = AlibiExplainRuntime(model_settings)
+    runtime.ready = await runtime.load()
+    return runtime, explainer, request, explain_kwargs
+
+
+async def test_end_2_end(test_case):
```

Comment:
I would just call it `test_explain` TBH, as that seems more clear (i.e. it's just testing the `.explain()` method for whitebox sklearn explainers, and the scaffolding just trains the explainer, the model, etc.

Reply:
Renamed in 53ce0b02e7886cc82a75bc52af27da1516207c6b